src/datasources/cache/redis.cache.service.spec.ts

```diff
@@ -144,4 +145,24 @@ describe('RedisCacheService', () => {
     // Connection is reopened after this test execution
     redisClient = await redisClientFactory();
   });
+
+  it('Increments and gets the value of a key', async () => {
```

Comment:
I would suggest the same as my previous comment regarding a second test.

Reply:
Added in [5f7073a](https://github.com/safe-global/safe-client-gateway/pull/1225/commits/5f7073afaea1b5398057abebd6bdfbba2bc9141b)